org-roam-doctor.el

```diff
@@ -294,7 +294,9 @@ If CHECKALL, run the check for all Org-roam files."
 (defun org-roam-doctor-start (files checkers)
   "Lint FILES using CHECKERS."
   (save-window-excursion
-    (let ((existing-buffers (org-roam--get-roam-buffers)))
+    (let ((existing-buffers (org-roam--get-roam-buffers))
+          (org-startup-with-latex-preview nil)
```

Comment:
Looks like we can have a variable `org-roam-doctor-inhibit-startup` that we set `org-inhibit-startup` to. Similar to how `org-agenda` speeds up the buffer preparation.

Reply:
https://github.com/org-roam/org-roam/pull/1057/commits/b2de9f4b76423010aa1c8c2a00bb7d05e7fac9b3